plugins/maciVoting/contexts/CoordinatorContext.tsx

```diff
@@ -97,25 +97,37 @@ export const CoordinatorProvider = ({ children }: { children: ReactNode }) => {
     [signer]
   );
 
+  const checkIsTallied = useCallback(
```

Comment:
added this btw: https://github.com/privacy-scaling-explorations/maci/blob/dev/packages/sdk/ts/tally/results.ts#L56